docs/endpoints/getGJLevelScores211.md

```diff
@@ -0,0 +1,49 @@
+# getGJLevelScores211.php
+
+Upload and query user levelBoard record.
+
+## Parameters
+
+**gameVersion** - The version of the game
+
+**binaryVersion** - The binary verion of the game
+
+**gdw** -- is geometry dash world
+
+**accountID** - The ID of the account
+
+**gjp** - The password encoded of the account
+
+**levelID** - The ID of the level
+
+**percent** - Percent of the player best
+
+**secret** - Wmfd2893gb7
+
+**type** - 0: friends, 1: top, 2: week
+
+**s1** - attempts + 8354
+
+**s2** - jumps + 3991
+
+**s3** - seconds + 4085
+
+**s4** - 1482 * 2 + (jumps + 3991) * (percentage + 8354) + ((seconds + 4085) ** 2) - 50028039
+
+**s4** - is 1482 if jumps, percentage and seconds are 0
```

Comment:
and what if they aren't?

Reply:
What I wanted to say here, when adding a comment to an issue, is that `s4` *follows* the formula while any of these values are non-zero. You can probably generalize that to `1482 * (has_played + 1)`, where `has_played` is a bool.

Reply:
ye maybe make that clear because this doesn't explain that at all

Reply:
I think this formula should be good then: `s4 = 1482 * (has_played + 1) + (jumps + 3991) * (percentage + 8354) + ((seconds + 4085) ^ 2) - 50028039`

Reply:
use spaces not snake case
again, we ain't using code blocks so stop treating it like it should be directly copied into someone's codes

these are docs. people need to read what something does
not have random places code snippets while other lines are just descriptions

Reply:
Understood. I think this formula should be highlighted in some way.

Reply:
` `` ` exists

Reply:
just highlight example calculations and add a description in front of them